compiler/rustc_const_eval/src/interpret/operand.rs

```diff
@@ -601,7 +601,9 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         // that the `RevealAll` pass has happened and that the body's consts
         // are normalized, so any call to resolve before that needs to be
         // manually normalized.
-        let val = self.tcx.normalize_erasing_regions(self.param_env, *val);
+        let Ok(val) = self.tcx.try_normalize_erasing_regions(self.param_env, *val) else {
+            throw_unsup!(Normalization)
```

Comment:
This is more invalid than unsupported I think.

Reply:
`throw_inval!(TooGeneric)` may be better suited then.

Reply:
Ah right, we already have that variant. Agreed.